docs/visual/e2e-testing/code-examples.md

```diff
@@ -9,7 +9,7 @@ import TabItem from '@theme/TabItem';
 
 To view Visual E2E code examples, please refer to the [Sauce Labs Training resources GitHub repo](https://github.com/saucelabs-training):
 
-* [JavaScript (WebDriverIO)](https://github.com/saucelabs-training/demo-js/tree/main/visual-e2e/examples)
+* [JavaScript (WebDriverIO)](https://github.com/saucelabs-training/demo-js/tree/main/webdriverio/webdriver/examples/visual-e2e)
 * [Java](https://github.com/saucelabs-training/demo-java/blob/main/selenium-junit4-examples/src/test/java/com/saucedemo/selenium/junit4/SimpleVisualE2ETest.java)
 * [Python](https://github.com/saucelabs-training/demo-python/tree/main/examples/sauce_visual)
 * [Ruby](https://github.com/saucelabs-training/demo-ruby/tree/main/sauce-features/visual/basic)
```

Comment:
```suggestion
* [Ruby](https://github.com/saucelabs-training/demo-ruby/tree/main/sauce-features/visual)
```

Reply:
Hi @mattdsauce , thanks again for submitting this PR! If this additional change looks ok to you, I'll go ahead and merge.